samples/reflection/bfbs2json.c

```diff
@@ -81,9 +81,15 @@ void print_object(reflection_Object_table_t O)
     reflection_Field_vec_t Flds;
     reflection_Field_table_t F;
     size_t i;
+    const char *name;
 
     Flds = reflection_Object_fields(O);
-    printf("{\"name\":\"%s\"", reflection_Object_name(O));
+
```

Comment:
This is why IBM added a slash or a dot to their zeroes. The argument is the letter following P in uppercase, not a zero, or it should have been. but I can't really tell. This is just a stupid argument. What happens if you change O to Obj?

Certainly your fix is safer, but unnecessary because a valid input buffer cannot have a null name, and if the buffer is not valid, there are 700 other places to crash harder than here.

Reply:
> What happens if you change O to Obj?
```
flatcc/samples/reflection/bfbs2json.c:86:55: error: ‘Obj’ undeclared (first use in this function)
   86 |     printf("{\"name\":\"%s\"", reflection_Object_name(Obj));
```

Reply:
You need to change the argument name of the function as well, and the other places where it is being used in the function.